R/tag_query.R

```diff
@@ -377,14 +377,14 @@ print.htmltools.tag.query <- function(x, ...) {
 format.htmltools.tag.query <- function(x, ...) {
   stop(
     "`format.htmltools.tag.query(x)` not allowed.\n",
-    "Please call `format()` the result of `$asTags()`"
+    "Please call `format()` the result of `$selected()`"
   )
 }
 #' @export
 as.character.htmltools.tag.query <- function(x, ...) {
   stop(
     "`as.character.htmltools.tag.query(x)` not allowed.\n",
-    "Please call `as.character()` the result of `$asTags()`"
+    "Please call `as.character()` the result of `$selected()`"
```

Comment:
"`tagQuery()` objects can not be written directly as HTML tags.\n", 
    "Call either `$root()` or `$selected()` to extract the tags of interest."